### ISSUE TITLE
Test descriptions Markdown should be rendered in failure section

### ISSUE THREAD

**Issue by darrenburns** [darrenburns/ward#246]
When we output a test with the `test-per-line` output mode, the Markdown renders. 

However, if that test fails, the Markdown doesn't render inside the `Rule`:

<img width="850" alt="Screenshot 2021-05-30 at 23 35 50" src="https://user-images.githubusercontent.com/5740731/120122334-18353380-c1a0-11eb-8f47-19632f88798e.png">

**Comment by carlosmarin96** [comment]
Hello! Could I give a try to this issue?

**Comment by darrenburns** [comment]
@carlosmarin96 Of course. I suspect it will involve moving from the Text class to the Markdown class. Let me know if I can help at all 🙂

**Comment by carlosmarin96** [comment]
@darrenburns Hi, sorry, I have problem. I'cant find the Text class :(

**Comment by darrenburns** [comment]
Hey @carlosmarin96, the `Text` and `Markdown` classes are part of the Rich library. They're not defined in the Ward codebase.

**Comment by AABur** [comment]
Simply changing the class from [Text](https://rich.readthedocs.io/en/latest/reference/text.html#rich.text.Text) to [Markdown](https://rich.readthedocs.io/en/latest/reference/markdown.html#rich.markdown.Markdown) won't help.
Parameter `title` of the [Rule](https://rich.readthedocs.io/en/latest/reference/rule.html?highlight=rule#rich.rule.Rule) class has the type `str | Text`.
Argument of type `Markdown` cannot be assigned to parameter `title` of type `str | Text`.
And it is not possible to convert `Markdown` to `Text` because Rich renders Markdown on the fly.

Alternatives:
- reject the idea completely and leave it as it is 🥇 
- display `test.description` under the rule with Markdown formatting and leave the rule as a line with no text
- output as it is, as a line with text inside, but remove the Markdown markup from that text 👎
